binr/radare2/radare2.c

```diff
@@ -1191,17 +1191,20 @@ int main(int argc, char **argv, char **envp) {
 			/* load symbols when doing r2 -d ls */
 			// NOTE: the baddr is redefined to support PIE/ASLR
 			baddr = r_debug_get_baddr (r.dbg, pfile);
-			if (baddr != UT64_MAX && baddr != 0) {
+			char *env_debug = r_sys_getenv ("R_DEBUG");
```

Comment:
`env_debug` should be freed

Reply:
but if you want to go for an env var at least dont reuse one that have a different meaning. use for example `R_DEBUG_DEBUGGER` in radare2.c to set the cfg.verbose variable